src/relentless/model/potential/__init__.py

```diff
@@ -29,6 +29,7 @@
 
 """
 
+from .bond import Harmonic
```

Comment:
We should be careful about this naming and import since we will quickly run into an issue when there are also harmonic angles, harmonic dihedrals, etc. For the pairs, I followed the convention that I put `Pair` somewhere in the class name if it was ambiguous. So this would change to `HarmonicBond`, but `FENE` would stay `FENE`. (Side note: don't forget to import FENE here.)

The other option would be not to import the bonded potentials into this namespace. The options would be:

* `relentless.model.potential.bond.Harmonic` vs.
* `relentless.model.potential.HarmonicBond`

I prefer the second.